### PR TITLE
Build and check formatting using Github Actions

### PR DESCRIPTION
**Pull request by oscar-b**
An initial workflow which will build and ch;eck code formatting.

The formatting change of `package.json` is done so that changes from `yarn` doesn't change the indentation all the time.

### CODE REVIEW

**Comment by oscar-b** [comment]
This is what it looks like in my repo, I assume it's not enabled here automatically without being merged first.

https://github.com/oscar-b/de1/commit/9173711bd7b0425d628203b34610be6d4d3a0fd5/checks?check_suite_id=387678709

File: package.json [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+    "packages/*"
+  ],
+  "repository": "git@github.com:LukasBombach/sblendid.git",
+  "author": "Lukas Bombach <lukas.bombach@googlemail.com>",
+  "license": "MIT",
+  "scripts": {
+    "prettier": "prettier --check packages/de1/src/**/*.{js,ts} packages/de1-debugger/src/**/*.{js,ts,tsx}"
+  },
+  "devDependencies": {
+    "prettier": "^1.19.1"
```
Comment (LukasBombach) [inline package.json]:
```suggestion
    "prettier": "1.19.1"
```

**Review by LukasBombach — CHANGES_REQUESTED** [review]
This is splendid! ;)

But I'd vote for no prettier customization whatsoever

**Comment by oscar-b** [comment]
@LukasBombach Could move the prettier settings to package.json if you prefer that.

**Comment by LukasBombach** [comment]
Nono, I'm actually in favor of separate files because the `package.json` might blow up at some point, I was just trying to escape formatting-discussion-hell (and a little bit the tabs-vs-spaces) discussion by just leaving the choices to someone else 😱 😆

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "useTabs": false,
+  "tabWidth": 2
+}
```
Comment (LukasBombach) [inline .prettierrc.json:6]: Those 3 already are the default values, but I wouldn't mind having trailing commas here if you prefer

**Review by LukasBombach — CHANGES_REQUESTED** [review]

File: .prettierrc.json (hunk 2 above)
Comment (oscar-b) [reply .prettierrc.json:6]: Added `trailingComma` to make git diffs smaller and moving lines easier :)

Comment (LukasBombach) [reply .prettierrc.json:6]: But you can remove the other 3 lines as they are already the default values: https://prettier.io/docs/en/options.html

**Review by LukasBombach — APPROVED** [review]

**Comment by LukasBombach** [comment]
😍 great work!

**Pull request merged**